doc/man/man3/advanced/crypto_hchacha20.3monocypher

```diff
@@ -114,7 +114,6 @@ crypto_wipe(in , 16);
 .Ed
 .Sh SEE ALSO
 .Xr crypto_chacha20_encrypt 3monocypher ,
```

Comment:
This one should be replaced by `crypto_chacha20` in a separate commit. (`crypto_chacha20_encrypt` is deprecated, and should not be mentioned here).